### PR TITLE
Search bar

### PR DESCRIPTION
**Pull request by devonbNU**
Implements search bar on recipe page. Currently assumes that recipe list is just a ul with each li containing only the name of the recipe

### CODE REVIEW

File: search-bar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+function applySearch(listName) {
+	let searchBox, filter, list, currentItem
+	searchBox = document.getElementById("search")
+	filter = searchBox.value.toUpperCase()
+	list = Array.from(document.getElementById(listName).getElementsByTagName('li'))
+	
+	list.forEach(listItem => {
+		titleElement = listItem // this will need to pull something from listItem once recipe list items implemented
```
Comment (jholm117) [inline search-bar.js]: titleElement is never defined

File: search-bar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function applySearch(listName) {
+	let searchBox, filter, list, currentItem
```
Comment (jholm117) [inline search-bar.js]: it looks like these are never reassigned so you should use const

File: search-bar.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	list.forEach(listItem => {
+		titleElement = listItem // this will need to pull something from listItem once recipe list items implemented
+		if (titleElement.innerHTML.toUpperCase().indexOf(filter) > -1)
+		{
+			listItem.style.display = ""
+		}
+		else
+		{
+			listItem.style.display = "none"
+		}
```
Comment (jholm117) [inline search-bar.js]: this is fine but I have basically the same code in recipe.js so we should extract this into a utility file at some point and reuse code. but lets worry about that later

File: search-bar.js (hunk 1 above)
Comment (devonbNU) [reply search-bar.js]: What do you mean by this? I don't declare it before in the let?

File: search-bar.js (hunk 2 above)
Comment (devonbNU) [reply search-bar.js]: For some reason making these const breaks it

Comment (jholm117) [reply search-bar.js]: you have to declare and assign on the same line i think so
`const searchBox = document.getElementById("search")`
as opposed to doing 
`const searchBox`
`searchBox = ...`

File: search-bar.js (hunk 1 above)
Comment (jholm117) [reply search-bar.js]: no, i don't see it anywhere

File: finder-stylesheet.css [hunk 4]
```diff
@@ -3,10 +3,14 @@ body{
...
-    display: block;
+    
+	display: block;
     margin: auto;
-    width: 30%;
+    width: 90%;
     height: 3%;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    padding-left: 5px;
```
Comment (jholm117) [inline finder-stylesheet.css]: It's not worth going back and changing this but moving forward i think we should try to stick with relative sizes e.g. rem or percentages

**Pull request merged**